packages/frontend-v2/utils/course/getCourseDisplayString.ts

```diff
@@ -1,9 +1,15 @@
-import { ScheduleCourse2 } from "@graduate/common";
+/** Only the details we need to display a course. */
+type CourseDisplayDetails = {
```

Comment:
This type is identical to CourseEqualDetails, so we might want to make it just one type in our types file that we use for both (and for potential future uses as well).

Reply:
Considered it but that couples the implementation of both functions. The types happen to be the same because we compare courses by the same stuff we use to generate their string. This could very well change in the future